jekyll-docs/datastores.md

```diff
@@ -36,7 +36,7 @@ ds1volume@datastore1
 We can check that volume is created in the datastore1
 
 ```
-$esx ls /vmfs/volumes/datastore1/dockvols
+$esx ls /vmfs/volumes/datastore1/ds1volume
```

Comment:
I think we should spell out the default tenant or a place holder in the path:

/vmfs/volumes/datastore1/dockvols/_DEFAULT/ds1volume
or 
/vmfs/volumes/datastore1/dockvols/<VMGROUP_NAME>/ds1volume